rules-reviewed/openshift/logging.windup.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0"?>
+<ruleset id="logging"
+    xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This is a ruleset for logging-related topics when migrating and application to OpenShift environments.
+        </description>
+        <dependencies>
+          <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final"/>
+          <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="java"/>
+        <sourceTechnology id="java-ee"/>
+        <targetTechnology id="openshift"/>
+        <targetTechnology id="cloud-readiness"/>
+        <tag>logging</tag>
+    </metadata>
+    <rules>
+        <rule id="logging-0000">
+            <when>
+                <javaclass references="org.apache.{*}log4j.{*}FileAppender{*}">
+                  <location>CONSTRUCTOR_CALL</location>
+                  <location>IMPORT</location>
+                  <location>INHERITANCE</location>
+                  <location>VARIABLE_DECLARATION</location>
+                </javaclass>
+            </when>
+            <perform>
+                <iteration>
+                    <hint title="Java Logging: Log4j configuration file" effort="3" category-id="openshift-optional">
```

Comment:
fyi category exists ;) in https://github.com/mrizzi/windup-rulesets/blob/fdbe0afaee2a60c0140b0f4418e4b148e09ab84e/rules-reviewed/core.windup.categories.xml#L38-L41